scripts/magpietts/convert_nemo_to_lhotse_shar.py

```diff
@@ -0,0 +1,464 @@
+# Copyright (c) 2025, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
```

Comment:
Can you add a readme? Not necessarily in this file. Could be elsewhere.
In it, can you
- Describe how to add a new set of codec files?
- Describe how to create a lhotse shar from previous datasets?

Reply:
sure, i can add a readme about how to create lhotse shar for magpietts. But for the guidance how to append new codec shards to the existing shards, I haven't tried it yet. In genenral, the solution is to load exiting cut shards, and attach the new codec codes to the MonoCut's custom field and write into shard files. This should uses the same index of shards.